pkg/runner/flags.go

```diff
@@ -61,6 +62,7 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	tenancy.ParseFlags(fs, &cfg.TenancyCfg)
 
 	fs.StringVar(&cfg.ConfigFile, "config", "config.yml", "YAML configuration file path for Promscale.")
+	fs.StringVar(&cfg.DatasetConfigFile, "startup.dataset.config", "", "Dataset YAML configuration file path for Promscale. It is used for setting various dataset configuration like default metric chunk interval")
```

Comment:
Why do we need 2 configuration files and not just a new section in existing config file?

Reply:
Discussed in the DD, this is the dataset configuration file which will be less structured than the main config file and will have different behavior (resetting all dataset configuration options to default if unset).